sdk/storage/storage-blob/swagger/README.md

```diff
@@ -78,6 +78,182 @@ directive:
     }
 ```
 
+
+### Rename maxresults -> maxResults
+``` yaml
+directive:
+- from: swagger-document
+  where: $.parameters.MaxResults
+  transform: >
+     $["name"] = "maxResults";
```

Comment:
just curious. what's the difference between using `name` v.s. `x-ms-client-name`?

Reply:
I think `x-ms-client-name` only overrides the name and no other properties such as serializedName. I think we want to use `x-ms-client-name`